### ISSUE TITLE
RuntimeError: asyncio.run() cannot be called from a running event loop

### ISSUE THREAD

**Issue by amaarora** [pydantic/pydantic-ai#122]
```python
from pydantic_ai import Agent
import dotenv

dotenv.load_dotenv()

agent = Agent(  
    'gemini-1.5-flash',
    system_prompt='Be concise, reply with one sentence.',  
)

import asyncio

result = agent.run_sync('Where does "hello world" come from?')  
```

>> RuntimeError: asyncio.run() cannot be called from a running event loop

**Comment by samuelcolvin** [comment]
Ye, I'm aware of this, already working to fix it.

**Comment by batrlatom** [comment]
this is stupid solution for sure, but i found a solution by using:

import nest_asyncio
nest_asyncio.apply()

**Comment by samuelcolvin** [comment]
This is fixed in #122, I'll make a release later today.
